src/cloud_provider/digitalocean/kubernetes/helm_charts.rs

```diff
@@ -428,15 +428,8 @@ pub fn do_helm_charts(
         chart_info: ChartInfo {
             name: "metrics-server".to_string(),
             path: chart_path("common/charts/metrics-server"),
+            values_files: vec![chart_path("chart_values/metrics-server.yaml")],
             values: vec![
-                ChartSetValue {
```

Comment:
why did you remove it?

Reply:
Because it was inconsistent. Replaced for all by a values file. If you prefer I can put it all in rust instead of yaml.